### PR TITLE
DSE 6.8.42 release notes

### PR DESCRIPTION
**Pull request by roxananeo**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

**Review by szymon-miezal — COMMENTED** [review]
I can see that the current state of this pull request contains a list of third-party software but it misses the release notes information itself.

File: DSE_6.8_Release_Notes.md [hunk 1]
```diff
@@ -4,6 +4,34 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
...
+
+## 6.8.42 DSE DSEFS
+* Fixed DSEFS file path handling that could fail when using filenames containing colons. (DSP-23947)
+
+## 6.8.42 DSE Insights
+* Removed Python 2.7 libraries from `collectd`. (DSP-23764)
+
+## 6.8.42 DSE CVE
+* Upgraded the DSE 6.8 dependency on Ehcache to Terracotta's version of Ehcache v2.10.10.17.20. The Terracotta version does not include extra libraries (specifically Jackson data-bind).  The previous Ehcache v2.10.9.2 was exposing a security vulnerability CVE-2020-36518.  The vulnerability in `jackson-databind` before v2.13.0 allowed a Java StackOverflow exception and denial of service via a large depth of nested objects. (DSP-23508, [CVE-2020-36518](https://nvd.nist.gov/vuln/detail/CVE-2020-36518), [CVE-2017-17485](https://nvd.nist.gov/vuln/detail/CVE-2017-17485), [CVE-2017-7525](https://nvd.nist.gov/vuln/detail/CVE-2017-7525), [CVE-2018-11307](https://nvd.nist.gov/vuln/detail/CVE-2018-11307), [CVE-2018-7489](https://nvd.nist.gov/vuln/detail/CVE-2018-7489), [CVE-2019-16942](https://nvd.nist.gov/vuln/detail/CVE-2019-16942))
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:33]:
```suggestion
* Upgraded the DSE 6.8 dependency on Ehcache to Terracotta's version of Ehcache v2.10.10.17.20. The Terracotta version does not include extra libraries (specifically Jackson databind). The previous Ehcache v2.10.9.2 was exposing a security vulnerability CVE-2020-36518. The vulnerability in `jackson-databind` before v2.13.0 allowed a Java StackOverflow exception and denial of service via a large depth of nested objects. (DSP-23508, [CVE-2020-36518](https://nvd.nist.gov/vuln/detail/CVE-2020-36518), [CVE-2017-17485](https://nvd.nist.gov/vuln/detail/CVE-2017-17485), [CVE-2017-7525](https://nvd.nist.gov/vuln/detail/CVE-2017-7525), [CVE-2018-11307](https://nvd.nist.gov/vuln/detail/CVE-2018-11307), [CVE-2018-7489](https://nvd.nist.gov/vuln/detail/CVE-2018-7489), [CVE-2019-16942](https://nvd.nist.gov/vuln/detail/CVE-2019-16942))
```

**Review by szymon-miezal — APPROVED** [review]
It looks good now.

File: DSE_6.8_Release_Notes.md (hunk 1 above)
Comment (roxananeo) [reply DSE_6.8_Release_Notes.md]: fixed

**Review by tiagomlalves — APPROVED** [review]

**Pull request merged**